### ISSUE TITLE
 FirebaseJobDispatcher (FJD) is depracted for android

### ISSUE THREAD

**Issue by boaz-lachman** [vikeri/react-native-background-job#144]
Hi,

I recently received an email from google:

tl:dr - FirebaseJobDispatcher is getting depracted and won't work on newer models of android

Could you please help?

"Hello Google Play Developer,

With the introduction of Jetpack WorkManager, we’re focusing on offering you a single go-to solution to background job scheduling on Android and are excited about the improvements in stability and the features it will bring. Read more in our recent blog post.

Starting November 1, 2020, GCMNetworkManager client libraries will be deprecated and no longer supported.

Note: FirebaseJobDispatcher (FJD) relies on GcmNetworkManager libraries and has already been deprecated.

To confirm that you currently do use FJD / GcmNetworkManager, please inspect your project code:

FJD:
com.firebase:firebase-jobdispatcher in your project dependencies
GcmNetworkManager
com.google.android.gms:play-services-gcm in your project dependencies AND
Use of com.google.android.gms.gcm.GcmNetworkManager in your code.
Once your app targets a future version of Android (> Android 10), GCMNetworkManager API calls via GcmNetworkManager or FJD client libraries will no longer work on devices running Android M and above.

Action required:

Please migrate to Jetpack WorkManager for your background job scheduling needs. We understand that code changes will be required, and we encourage you to factor in time for adequate testing as well."
